javascript/javascript1/week2/task_2.js

```diff
@@ -0,0 +1,32 @@
+//array for names of days of week
+const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+
+function getEventWeekday(after_x_days_of_today) {
```

Comment:
In JavaScript developers usually give the variables names as [camel case](https://en.wikipedia.org/wiki/Camel_case).